taskt/UI/Forms/ThemedForm.cs

```diff
@@ -26,27 +22,24 @@ private void ThemedForm_Load(object sender, EventArgs e)
         protected override void OnShown(EventArgs e)
         {
             base.OnShown(e);
-            if (this.ClientSize.Height > Screen.PrimaryScreen.WorkingArea.Height - this.CurrentAutoScaleDimensions.Height) // Resizes if too tall to fit
+            // Resizes if too tall to fit
+            if (ClientSize.Height > Screen.PrimaryScreen.WorkingArea.Height - CurrentAutoScaleDimensions.Height)
             {
-                this.ClientSize = new Size(this.ClientSize.Width, Screen.PrimaryScreen.WorkingArea.Height - (int)this.CurrentAutoScaleDimensions.Height);
+                ClientSize = new Size(ClientSize.Width, Screen.PrimaryScreen.WorkingArea.Height -
+                    (int)CurrentAutoScaleDimensions.Height);
             }
         }
 
-        private taskt.Core.Theme _Theme = new taskt.Core.Theme();
-        public taskt.Core.Theme Theme
-        {
-            get { return _Theme; }
-            set { _Theme = value; }
-        }
-
         protected override void OnPaint(PaintEventArgs e)
         {
-            e.Graphics.FillRectangle(this.Theme.CreateGradient(this.ClientRectangle), this.ClientRectangle);
+            e.Graphics.FillRectangle(Theme.CreateGradient(ClientRectangle), ClientRectangle);
             base.OnPaint(e);
         }
+
         public static void MoveFormToBottomRight(Form sender)
         {
-            sender.Location = new Point(Screen.FromPoint(sender.Location).WorkingArea.Right - sender.Width, Screen.FromPoint(sender.Location).WorkingArea.Bottom - sender.Height);
+            sender.Location = new Point(Screen.FromPoint(sender.Location).WorkingArea.Right -
```

Comment:
This is really hard to read, I'd suggest breaking it up by making new variables for the two method arguments. Generally string concatenation and methods are easy enough to read even when broken across multiple lines but when math operations start to become intermingled it leads to a much harder to parse statement.

This is kind of what I would suggest:
```
var x = Screen.FromPoint(sender.Location).WorkingArea.Right - sender.Width
var y = Screen.FromPoint(sender.Location).WorkingArea.Bottom - sender.Height

sender.Location = new Point(x,y)
```

Reply:
done